addOns/grpc/src/main/java/org/zaproxy/addon/grpc/internal/VariantGrpc.java

```diff
@@ -51,7 +51,7 @@ public void setMessage(HttpMessage msg) {
                 protoBufMessageDecoder.decode(payload);
                 parseContent(protoBufMessageDecoder.getDecodedToList(), "");
                 requestDecodedBody = protoBufMessageDecoder.getDecodedOutput();
-            } catch (InvalidProtobufFormatException e) {
+            } catch (InvalidProtobufFormatException | IllegalArgumentException e) {
                 LOGGER.error("Parsing message body failed: {}", e.getMessage());
```

Comment:
Is the `IllegalArgumentException` because of the empty body? If so that should be just debug, although `isValidContentType` could be changed to `isValidGrpcMessage` and return false if the body is empty.

Reply:
actually IllegalArgumentException occur when wire type say it is string but then we try to parse it , but it not valid utf-8 so give this exception ?